packages/webview_flutter/tizen/src/webview.cc

```diff
@@ -171,31 +171,34 @@ std::string WebView::GetNavigationDelegateChannelName() {
 }
 
 void WebView::Dispose() {
-  texture_registrar_->UnregisterTexture(GetTextureId(), nullptr);
-
-  if (webview_instance_) {
-    evas_object_smart_callback_del(webview_instance_,
-                                   "offscreen,frame,rendered",
-                                   &WebView::OnFrameRendered);
-    evas_object_smart_callback_del(webview_instance_, "load,started",
-                                   &WebView::OnLoadStarted);
-    evas_object_smart_callback_del(webview_instance_, "load,finished",
-                                   &WebView::OnLoadFinished);
-    evas_object_smart_callback_del(webview_instance_, "load,progress",
-                                   &WebView::OnProgress);
-    evas_object_smart_callback_del(webview_instance_, "load,error",
-                                   &WebView::OnLoadError);
-    evas_object_smart_callback_del(webview_instance_, "console,message",
-                                   &WebView::OnConsoleMessage);
-    evas_object_smart_callback_del(webview_instance_,
-                                   "policy,navigation,decide",
-                                   &WebView::OnNavigationPolicy);
-    evas_object_smart_callback_del(webview_instance_, "url,changed",
-                                   &WebView::OnUrlChange);
-    evas_object_del(webview_instance_);
-  }
+  if (!disposed_) {
```

Comment:
Or simply
```suggestion
  if (disposed_) {
    return;
  }
```
But is there any case where `Dispose` is called twice?

Reply:
Call `dispose()` on the `Webview` destructor, and also called through `OnDispose()` of `PlatformViewChannel`.
I can also delete `dispose()` of the destructor, but I think this way frees memory more clearly.